proto/p4/config/v1/p4info.proto

```diff
@@ -48,6 +48,20 @@ message Documentation {
   string description = 2;
 }
 
+// Used to describe the required properties of the underlying platform.
+message PlatformProperties {
+  // The minimum number of multicast entries (i.e. multicast groups) that the
+  // platform is required to support. If 0, there are no requirements.
+  int64 multicast_group_table_size = 1;
```

Comment:
Any particular reason for using `int64` over `int32` here (and below)?

Reply:
Good question! Simply to be symmetric to tables and action profiles where `size` is an `int64` (though `max_group_size` is an `int32`, so I should at least update that). Probably. I actually don't totally remember.

That said, [`multicast_group_id` is `uint32`](https://github.com/p4lang/p4runtime/blob/main/proto/p4/v1/p4runtime.proto#L452), so it's a bit questionable to have the size be `int64`... 

I'm convinced! Changed it all to `int32`.